client/src/app/shared/shared-main/users/user-notifications.component.ts

```diff
@@ -39,18 +40,25 @@ export class UserNotificationsComponent implements OnInit {
       totalItems: null
     }
 
-    this.loadMoreNotifications()
+    this.loadNotifications()
 
     if (this.markAllAsReadSubject) {
       this.markAllAsReadSubject.subscribe(() => this.markAllAsRead())
     }
   }
 
-  loadMoreNotifications () {
-    this.userNotificationService.listMyNotifications(this.componentPagination, undefined, this.ignoreLoadingBar)
+  loadNotifications (reset?: boolean) {
+    this.userNotificationService.listMyNotifications({
+      pagination: this.componentPagination,
+      ignoreLoadingBar: this.ignoreLoadingBar,
+      sort: {
+        field: this.sortField,
+        order: this.sortField === 'createdAt' ? -1 : 1
```

Comment:
if `createdAt` then it's an ASC order no?

Reply:
we want to list notifications in `DESC` order, right?

Reply:
Yes but `-createdAt` is `DESC` order so `-1` and `createdAt` is `ASC` order so `1` :thinking:

Reply:
okay. something else must be wrong somewhere, then.

Reply:
@Chocobozzz WDYT?

Reply:
Oh okay then, I think I understood. LGTM, ready to merge

Reply:
I'll just add a comment